css/src/main.css

```diff
@@ -1165,6 +1165,8 @@ progress {
   .col-md-12 {
     flex: 0 0 auto;
     width: 100%;
+    justify-content:center;
```

Comment:
Rather than adding this rule to the Bootstrap source code, use the Bootstrap utilities `d-flex` and `justify-content-center` in the HTML on the necessary column.